packages/amplify-graphql-api-construct/src/__tests__/amplify-dynamodb-table-wrapper.test.ts

```diff
@@ -292,5 +292,16 @@ describe('AmplifyDynamoDbTable', () => {
         });
       });
     });
+
+    describe('applyRemovalPolicy', () => {
+      it('overrides the removal policy', () => {
+        testResource.applyRemovalPolicy(RemovalPolicy.DESTROY);
```

Comment:
1. Do we have tests to assert default values, or is that a relevant concept here?
2. Is there a removal policy getter on the wrapper? If so, does it store a separate property or proxy remoal policy of the underlying resource?

Reply:
1. We don't have the test. The default values are set in the resource generator of model transformer which is `DESTROY` be default. I can add the default value check in my follow import table PR as its default value is `RETAIN`. This unit test is for the setter only.
2. There is no getter for the wrapper. Technically, the removal policy is not a resource property but a resource option as in the CFN template they are defined at the same level as `Properties`